deployment/hasura/migrations/Aerie/3_sequence_parcel/up.sql

```diff
@@ -0,0 +1,375 @@
+
+drop view sequencing.rule_expansion_set_view;
+
+create or replace  view sequencing.rule_expansion_set_view as
+select str.rule_id,
+       set.id,
+       set.name,
+       set.owner,
+       set.description,
+       set.parcel_id,
+       set.mission_model_id,
+       set.created_at,
+       set.updated_at,
+       set.updated_by
+from sequencing.expansion_set_to_rule str left join sequencing.expansion_set set
+                                                    on str.set_id = set.id;
+
+
+alter table sequencing.expansion_set
+  drop column command_dict_id;
+
+--------------------------------------
+---- Modify the Expansion Rule Table
+--------------------------------------
+
+alter table sequencing.expansion_rule
+  add column parcel_id integer,
+  add foreign key (parcel_id)
+    references sequencing.parcel (id)
+    on delete set null;
+
+comment on column sequencing.expansion_rule.parcel_id is e''
+  'The id of the parcel to be used for authoring of this expansion.';
+
+--- Data Migration
+
+update sequencing.expansion_rule sr
+set parcel_id = p.id
+from sequencing.parcel p
+where p.command_dictionary_id = sr.authoring_command_dict_id;
+
+---
+
+alter table sequencing.expansion_rule
+  add constraint expansion_rule_unique_name_per_parcel_and_model
+    unique (authoring_mission_model_id, parcel_id, name),
+
+  drop constraint expansion_rule_unique_name_per_dict_and_model,
+  drop constraint expansion_rule_authoring_command_dict_id_fkey;
+
+drop view sequencing.expansion_set_rule_view;
+
+create or replace view sequencing.expansion_set_rule_view as
+select str.set_id,
+       rule.id,
+       rule.activity_type,
+       rule.expansion_logic,
+       rule.parcel_id,
+       rule.authoring_mission_model_id,
+       rule.created_at,
+       rule.updated_at,
+       rule.name,
+       rule.owner,
+       rule.updated_by,
+       rule.description
+from sequencing.expansion_set_to_rule str
+left join sequencing.expansion_rule rule
+on str.rule_id = rule.id;
+
+alter table sequencing.expansion_rule
+  drop column authoring_command_dict_id;
+
+
+---------------------------------------
+--- Modify the Command Dictionary Table
+---------------------------------------
+
+alter table sequencing.command_dictionary
+  add column dictionary_path text,
+  add column updated_at timestamptz not null default now();
+
+comment on column sequencing.command_dictionary.dictionary_path is e''
+  'The location of command dictionary types (.ts) on the filesystem';
+
+create trigger set_timestamp
+  before update on sequencing.command_dictionary
+  for each row
+execute function util_functions.set_updated_at();
+
+--- Data Migration
+update sequencing.command_dictionary
+set dictionary_path = command_types_typescript_path
+where dictionary_path is null;
+
+alter table sequencing.command_dictionary
+  alter column dictionary_path set not null;
+
+---
+
+alter table sequencing.command_dictionary
+  drop column command_types_typescript_path;
```

Comment:
This part of the migration can be simplified since it is just a column rename and add:
```sql
---------------------------------------
--- Modify the Command Dictionary Table
---------------------------------------
alter table sequencing.command_dictionary
  add column updated_at timestamptz not null default now();

alter table sequencing.command_dictionary
 rename column command_types_typescript_path to dictionary_path;

create trigger set_timestamp
  before update on sequencing.command_dictionary
  for each row
execute function util_functions.set_updated_at();
```

Reply:
similar question about the `comment` do I still need to create one if we do a column rename?